cra-network-aware-data-fetching/src/App.js

```diff
@@ -1,26 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
+import IFixitSearch from './containers/IFixitSearch';
```

Comment:
Can we change this to iFixitSearch?

Reply:
I'm afraid it looks like it's sort of rules in React component naming.
https://www.dropbox.com/s/7kc6g9x0lm9wf49/2.PNG?dl=0

Reply:
Ah. Not a huge deal then :)